bucket/manifold-viewer-9-official.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "9.0.174",
+    "description": "Official build of Manifold Viewer Release 9",
+    "homepage": "http://manifold.net/",
+    "license": "Unknown",
+    "depends": "extras/vcredist2017" ,
+    "url": "http://www.manifoldgis.com/updates/manifold-viewer-9.0.174-x64.zip",
```

Comment:
This is clearly intended as 64bit only

Reply:
well, this really includes both versions. Running 32bit version might be needed even on 64bit machine for compatibility with other software http://www.manifold.net/doc/mfd9/launch_in_32-bit_mode.htm

Reply:
Oh, should I put this url inside   "architecture": {  "64bit": { }}  ?